content/flux/v0.x/stdlib/influxdata/influxdb/from.md

```diff
@@ -90,11 +90,48 @@ If authentication is _disabled_, provide an empty string (`""`).
 If authentication is _enabled_, provide your InfluxDB username and password
 using the `<username>:<password>` syntax.
 
+## Push down optimizations
+
+Some transformations after the call to `from()` will trigger performance optimizations called push downs.
+These optimizations are "pushed down" from Flux into the storage layer and involve utilizing some code in storage to directly apply the transformation to the data to either filter or aggregate the data.
+These happen automatically and the person writing the query doesn't have to know about the technical details, but it is helpful to know about these optimizations so you can understand why certain orders of transformations may be faster than others.
+
+Push downs require the chain between transformations to be unbroken and exclusive.
+A `from()` that goes to multiple push down optimizations will cause none of them to be applied.
+To avoid this while allowing for code reuse, prefer invoking `from()` in a function when multiple push downs get applied.
+
+### Filter
+
+A `filter()` that makes a comparison to `r._measurement`, `r._field`, `r._value` or any tag value will be pushed down to the storage layer.
+Comparisons that use functions will not.
+If the function produces a static value, you can evaluate the function outside of the filter and the result will be pushed down.
+Multiple consecutive filters that would get pushed down will get merged together into a single filter that gets pushed down.
+
+### Aggregates
+
+The following aggregates will be pushed down:
+
+- `min()`
+- `max()`
+- `sum()`
+- `count()`
+- `mean()` (except when used with `group()`)
+
+Aggregates will also be pushed down if they are preceded by a `group()`.
+The only exception is `mean()` which cannot be pushed down to the storage layer with `group()`.
```

Comment:
This list seems incomplete? It doesn't appear to include the work that Faith did and what we currently have in the documentation: https://docs.influxdata.com/influxdb/cloud/query-data/optimize-queries/#pushdown-functions-and-function-combinations

Reply:
Some of those functions are ones I'm not aware of being pushdowns. In particular, I've never seen rename as something that is pushed down. Duplicate is part of aggregate window so it is "pushed down" in that context, but I prefer not to over-complicate things by saying duplicate is pushed down.

I'll have to investigate. Should I adapt this document for the same page? I wasn't aware that one existed.